core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -764,10 +785,13 @@ private[spark] object SparkSubmitUtils {
       coordinates: String,
       remoteRepos: Option[String],
       ivyPath: Option[String],
-      isTest: Boolean = false): String = {
+      isTest: Boolean = false): Seq[String] = {
     if (coordinates == null || coordinates.trim.isEmpty) {
-      ""
+      Seq("")
```

Comment:
Did you mean to include a single element sequence? Or did you mean to do `Seq.empty`